polling_stations/apps/file_uploads/models.py

```diff
@@ -131,6 +135,47 @@ def pr_body(self):
         else:
             return f"**NB triggered from local machine**\n{message}"
 
+    def send_confirmation_email(self, user):
+        if user.email:
+            to = user.email
+        else:
+            to = settings.DEFAULT_FROM_EMAIL
+        subject = "Your file upload was successful"
+        message = render_to_string(
+            template_name="file_uploads/email/upload_confirmation.txt",
+            context={
+                "subject": subject,
+            },
+        )
+
+        email = EmailMessage(
+            subject,
+            message,
+            settings.DEFAULT_FROM_EMAIL,
+            [to],
+            [settings.DEFAULT_FROM_EMAIL],
+            reply_to=[settings.DEFAULT_FROM_EMAIL],
+            headers={"Message-ID": subject},
+        )
+
+        return email.send()
+
+    def send_error_email(self):
+        subject = "File upload failed"
+        message = (
+            "File upload for council {self.gss} failed. Please investigate further."
```

Comment:
Another thought, could we include the `timestamp` on here? Could just use `self.__str__`.  So that we know which upload we're talking about.